### PR TITLE
Add GitHub settings file

### PR DESCRIPTION
**Pull request by caniszczyk**
Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: toc
+
+  # A short description of the repository that will show up on GitHub
+  description: CDF TOC
+
+  # A URL with more information about the repository
+  homepage: https://cd.foundation
```
Comment (oleg-nenashev) [inline .github/settings.yml:11]:
```suggestion
  homepage: https://cd.foundation/about/toc/
```

File: .github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: toc
+
+  # A short description of the repository that will show up on GitHub
+  description: CDF TOC
```
Comment (oleg-nenashev) [inline .github/settings.yml:8]:
```suggestion
  description: CDF Technical Oversight Committee (TOC)
```

**Review by oleg-nenashev — APPROVED** [review]
It would require https://github.com/apps/settings to be installed for this repository, and it will reset manual settings if any. Otherwise +1 from me

**Comment by afrittoli** [comment]
Looking at https://github.com/apps/settings, this encompasses all settings in the repo, including labels, branch protection, milestones etc. It would be great if it was possible to export existing settings into a YAML to use a starting point for this.
I've looked a bit in https://github.com/probot/settings/tree/master but I haven't found a way to do so - but with some js script expertise it shouldn't be too hard.

The other thing worth noting is that using this app requires some [security configuration](https://github.com/probot/settings/tree/master#security-implications) to be done - it shouldn't be a problem though

File: .github/settings.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: toc
+
+  # A short description of the repository that will show up on GitHub
+  description: CDF Technical Oversight Committee (TOC)
```
Comment (oleg-nenashev) [inline .github/settings.yml:8]:
```suggestion
  description:  "CDF Technical Oversight Committee (TOC) facilitates communication and collaboration among the Technical Projects"
```

**Comment by oleg-nenashev** [comment]
Ready to merge now IMO

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Pull request merged**